Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/SettingsPage.xaml

```diff
@@ -11,43 +13,118 @@
+            <StackLayout Orientation="Vertical" VerticalOptions="Start">
+                <StackLayout Spacing="5" Orientation="Vertical" VerticalOptions="Start" Padding="10">
+                    <Label Text="License" FontAttributes="Bold" FontSize="Subtitle" TextColor="White"/>
+                    <Label>
+                        <Label.FormattedText>
+                            <FormattedString>
+                                <Span Text="This app is open source software licensed under the " TextColor="White" FontSize="Body"/>
+                                <Span Text="MIT License"
+                                    TextColor="White"
+                                    TextDecorations="Underline"
+                                    FontSize="Body">
+                                    <Span.GestureRecognizers>
+                                        <TapGestureRecognizer Command="{Binding OpenUrl}"
+                                            CommandParameter="https://mit-license.org/"/>
+                                    </Span.GestureRecognizers>
+                                </Span>
+                                <Span Text="." TextColor="White" FontSize="Body"/>
+                            </FormattedString>
+                        </Label.FormattedText>
+                    </Label>
+                </StackLayout>
+                <StackLayout Spacing="5" Orientation="Vertical" VerticalOptions="Start" Padding="10">
+                    <Label Text="Contribute" FontAttributes="Bold" FontSize="Subtitle" TextColor="White"/>
+                    <Label>
+                        <Label.FormattedText>
+                            <FormattedString>
+                                <Span Text="Go to our " TextColor="White" FontSize="Body"/>
+                                <Span Text="Github Repository"
+                                    TextColor="White"
+                                    TextDecorations="Underline"
+                                    FontSize="Body">
+                                    <Span.GestureRecognizers>
+                                        <TapGestureRecognizer Command="{Binding OpenUrl}"
+                                            CommandParameter="https://github.com/amosproj/amos2022ss05-find-my-hearing-aid" />
+                                    </Span.GestureRecognizers>
+                                </Span>
+                                <Span Text="." TextColor="White" FontSize="Body"/>
+                            </FormattedString>
+                        </Label.FormattedText>
+                    </Label>
+                </StackLayout>
+                <StackLayout Spacing="5" Orientation="Vertical" VerticalOptions="Start" Padding="10">
+                    <Label Text="Version" FontAttributes="Bold" FontSize="Subtitle" TextColor="White"/>
+                    <Label Text=
+                           "1.0.0"
```

Comment:
I just changed the version to "0.12.0" since our next release will be sprint-12-release. 
I think updating the version for every (sprint-)release, maybe even as part of the release manager's job, is a nice tradeoff between the version's meaningfulness and the required effort of keeping it up-to-date. 
In case you are completely unhappy with this compromise and want to change it before the deadline of 9 am, you are allowed to change it back with a commit directly on the dev branch, no pull-request required.